ProjectCoimbra.UWP/Project.Coimbra/Pages/BandSongPage.xaml.cs

```diff
@@ -33,6 +33,7 @@ public BandSongPage()
 
             this.SetCustomEventHandlers();
 
+            //TODO: Display a loading bar when initializing this BandSongPage
```

Comment:
Nitpick: `//` comments should begin with a space by convention.

But I'd favour not putting this here and just tracking it as a bug, as it's not 100% clear we'll get around to this given that priorities could change.

Reply:
IMO, I would still put it here, because for now it is an UX issue that we probably do not want to see it freezing when using LAN mode. And it is just a comment to tell people that the problem point is here, so it reduces the time for debugging. It is not a code change also.